### PR TITLE
meta tags for search results, fb twitter add, favicon add, twitter follow button add

### PR DESCRIPTION
**Pull request by maddhruv**
### Additions
* meta description of the site
* google bots meta tags add
* og tags for facebook share
* twitter card tags for share on twitter
* twitter follow button add

### Files addition
* fav png add

### CODE REVIEW

**Comment by gigabo** [comment]
Good idea @midhruvjaink!

Can probably generate a nicer favicon from https://github.com/lerna/logo.

File: index.html [hunk 1]
```diff
@@ -5,6 +5,28 @@
...
+    <meta content="1 days" name="revisit-after" />
+    <meta content="all" name="robots" />
+    <meta content="all" name="googlebot" />
+    <!--social links-->
+    <meta property="og:title" content="Lerna · A tool for managing JavaScript projects with multiple packages." />
+    <meta property="og:site_name" content="lernajs" />
+    <meta property="og:type" content="website" />
+    <meta property="og:description" content="A tool for managing JavaScript projects with multiple packages." />
+    <meta property="og:url" content="https://lernajs.io/" />
+    <meta property="og:image" content="images/lerna-hero.svg" />
```
Comment (gigabo) [inline index.html]: Might also be better to make this the logo rather than the hero image.

File: index.html [hunk 2]
```diff
@@ -5,6 +5,28 @@
...
+    <meta property="og:image:width" content="256" />
+    <meta property="og:image:height" content="256" />
+    <meta property="og:image:type" content="image/svg" />
+    <meta name="twitter:site" content="@lernajs" />
+    <meta name="twitter:creator" content="@lernajs" />
+    <meta name="twitter:title" content="Lerna" />
+    <meta name="twitter:description" content="A tool for managing JavaScript projects with multiple packages." />
+    <meta name="twitter:image" content="images/lerna-hero.svg" />
```
Comment (gigabo) [inline index.html]: And this.

File: index.html (hunk 1 above)
Comment (maddhruv) [reply index.html]: Okk I'll make the changes up in some time!!

**Comment by maddhruv** [comment]
hey @gigabo 
the favicon has been changed with addition of a new share image

We can merge the PR now, I hope!

**Comment by gigabo** [comment]
Nice @midhruvjaink thanks for the fast turnaround!

That favicon looks way better.  Probably as good as it'll get without hand-editing.

I guess the other question is should we use the cropped square like we have for the GitHub org instead of the full logo as a square.

As in this:

lerna-favicon

vs this:

image

File: index.html [hunk 3]
```diff
@@ -5,6 +5,28 @@
...
+    <meta content="1 days" name="revisit-after" />
+    <meta content="all" name="robots" />
+    <meta content="all" name="googlebot" />
+    <!--social links-->
+    <meta property="og:title" content="Lerna · A tool for managing JavaScript projects with multiple packages." />
+    <meta property="og:site_name" content="lernajs" />
+    <meta property="og:type" content="website" />
+    <meta property="og:description" content="A tool for managing JavaScript projects with multiple packages." />
+    <meta property="og:url" content="https://lernajs.io/" />
+    <meta property="og:image" content="images/share.png" />
```
Comment (gigabo) [inline index.html:21]: I'm not seeing `images/share.png` in the repo.  Did you `git add` it?

**Comment by maddhruv** [comment]
I have added the image now

Somehow I missed adding it!

**Comment by jamiebuilds** [comment]
> I guess the other question is should we use the cropped square like we have for the GitHub org instead of the full logo as a square.

I would crop it all the way down to a single head so it's recognizable

**Comment by maddhruv** [comment]
I think the images are quite too big to be used!

**Comment by maddhruv** [comment]
any update on this?

**Pull request closed without merging**